Source_Code/Client/ClueLessClient.java

```diff
@@ -362,6 +396,38 @@ else if( statUp instanceof SuggestionWrong)
         {
             System.out.println(((SuggestionWrong) statUp).RefuterName + " was able to refute the suggestion");
         }
+
+        // Notify players about an accusation
+        else if (statUp instanceof AccuseNotification)
```

Comment:
Been trying to offload these into separate functions like in the server because this function was getting huge

Reply:
Yeah appreciate keeping it clean. Moved.